Sources/SmileID/Classes/Networking/Metadata/AnyCodable.swift

```diff
@@ -0,0 +1,147 @@
+// swiftlint:disable all
+import Foundation
+/**
+ A type-erased `Codable` value.
+
+ The `AnyCodable` type forwards encoding and decoding responsibilities
+ to an underlying value, hiding its specific underlying type.
+
+ You can encode or decode mixed-type values in dictionaries
+ and other collections that require `Encodable` or `Decodable` conformance
+ by declaring their contained type to be `AnyCodable`.
+
+ - SeeAlso: `AnyEncodable`
+ - SeeAlso: `AnyDecodable`
+ */
+@frozen public struct AnyCodable: Codable {
```

Comment:
Does it make sense to move this file also to the Helpers section?

Reply:
Ah yes. I will move it there.